mempool/clist_mempool_test.go

```diff
@@ -748,14 +763,18 @@ func doCommit(t require.TestingT, mp Mempool, app abci.Application, txs types.Tx
 	for i, tx := range txs {
 		rfb.Txs[i] = tx
 	}
-	_, e := app.FinalizeBlock(context.Background(), rfb)
-	require.True(t, e == nil)
+	_, err := app.FinalizeBlock(context.Background(), rfb)
+	require.Nil(t, err)
+
 	mp.Lock()
-	e = mp.FlushAppConn()
-	require.True(t, e == nil)
-	_, e = app.Commit(context.Background(), &abci.RequestCommit{})
-	require.True(t, e == nil)
-	e = mp.Update(height, txs, abciResponses(txs.Len(), abci.CodeTypeOK), nil, nil)
-	require.True(t, e == nil)
-	mp.Unlock()
+	defer mp.Unlock()
+
+	err = mp.FlushAppConn()
+	require.Nil(t, err)
+
+	_, err = app.Commit(context.Background(), &abci.RequestCommit{})
+	require.Nil(t, err)
+
+	err = mp.Update(height, txs, abciResponses(txs.Len(), abci.CodeTypeOK), nil, nil)
+	require.Nil(t, err)
```

Comment:
I am not sure to understand the rationale of these changes. The test was written with `mempoolSize > cacheSize` in mind. Now, it works with  `mempoolSize >= cacheSize`, but still fails when`mempoolSize < cacheSize`.

Reply:
These changes in `doCommit` are just a small refactoring to make the code consistent with the rest of the testing code. They shouldn't modify the result of the tests.